go/gildedrose/gildedrose_test.go

```diff
@@ -0,0 +1,18 @@
+package gildedrose_test
+
+import (
+	"REPO_URL/YOUR-USER-OR-ORG-NAME/YOUR-REPO-NAME/gildedrose"
```

Comment:
do you want to replace REPO_URL/YOUR-USER etc with github.com/emilybache, or is there a good reason not to?

Reply:
Yes, I can change it, of course. I opted to indicate it as a template, but there is no other reason honestly.